R/synthetic_cdisc_data.R

```diff
@@ -104,10 +129,10 @@ ls_synthetic_cdisc_data <- function() {
 
     names(all) <- c("Name", "Title", "Package")
 
-    dates <- as.Date(substring(all$Name, nchar(all$Name) - 9), format = "%Y_%m_%d")
+    dates <- as.Date(substring(all$Name, 5, 14), format = "%Y_%m_%d")
 
     all$Latest <- FALSE # nolint
-    all$Latest[which.max(dates)] <- TRUE
+    all$Latest[dates == max(dates)] <- TRUE
```

Comment:
is this for cases with multiple values? why `which.max` is not ok here?

Reply:
`which.max` returns only the index of the first occurrence of the max. date. In this case, we want to label *all* of the individual datasets from the latest release as `Latest = TRUE`, so use `dates == max(dates)` which returns multiple indices instead.